rubrikcdm/cluster.go

```diff
@@ -869,3 +883,67 @@ func (c *Credentials) Bootstrap(clusterName, adminEmail, adminPassword, manageme
 
 	return bootstrap, nil
 }
+
+// RegisterCluster submits the registration details for the specified Rubrik cluster. The username and password should
+// correspond to your Rubrik Support Portal account.
+func (c *Credentials) RegisterCluster(username, password string, timeout ...int) (interface{}, error) {
+
+	httpTimeout := httpTimeout(timeout)
+
+	// Change the default to 160
+	if httpTimeout == 15 {
+		httpTimeout = 60
```

Comment:
@drew-russell which is correct, the comment (160) or the value assigned to httpTimeout (60)?